src/_pytest/debugging.py

```diff
@@ -77,9 +77,7 @@ def pytest_configure(config):
     if config.getvalue("usepdb"):
         config.pluginmanager.register(PdbInvoke(), "pdbinvoke")
 
-    pytestPDB._saved.append(
-        (pdb.set_trace, pytestPDB._pluginmanager, pytestPDB._config, pytestPDB._pdb_cls)
-    )
+    pytestPDB._saved.append(pdb.set_trace)
```

Comment:
I'm a little worried about this because this leaves `config` and `pluginmanager` alive after `pytest.main()`, which might be a problem for someone embedding `pytest` I guess.

I wonder if we can only pass what is is actually needed to `pytestPDB` instead of the entire `config` and `pluginmanager` objects?

Reply:
Well, the main point here is to keep them alive after pytest exits, isn't it?.. ;)

(The whole wrapping can be improved in general - I'm doing this basically step by step when running into issues)

Reply:
> Well, the main point here is to keep them alive after pytest exits, isn't it?.. ;)

Yep, and I'm not sure that's a good thing 😅, specially if I'm not using threads at all.

Reply:
An alternative would be to fix the debugging plugin to not crash in this case I think.